Tests/PackageLoadingTests/PackageBuilderTests.swift

```diff
@@ -930,6 +930,23 @@ class PackageBuilderTests: XCTestCase {
         }
     }
 
+    func testDuplicateTargets() throws {
```

Comment:
Please add similar test in `PackageBuilderV4Tests.swift`. We support two versions of `PackageDescription`, so its better to get coverage for both.